nannou/src/event.rs

```diff
@@ -1,9 +1,10 @@
 //! Application, event loop and window event definitions and implementations.
 //!
-//! - [**Event**](./enum.Event.html) - the defualt application event type.
-//! - [**winit::event::WindowEvent**](./struct.WindowEvent.html) - events related to a single window.
-//! - [**WindowEvent**](./struct.WindowEvent.html) - a stripped-back, simplified,
-//!   newcomer-friendly version of the **raw**, low-level winit event.
+//! - [**Event**](./enum.Event.html) - the default application event type.
+//! - [**winit::event::WindowEvent**](https://docs.rs/winit/0.22.1/winit/event/enum.WindowEvent.html) -
```

Comment:
Would you mind changing:

> https://docs.rs/winit/0.22.1/winit/event/enum.WindowEvent.html

to

> https://docs.rs/winit/latest/winit/event/enum.WindowEvent.html

This will ensure we are always pointing to the latest version, provided winit doesn't move the type again :sweat_smile: